CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# dbt_facebook_ads_source v0.9.0
+
+[PR #44](https://github.com/fivetran/dbt_facebook_ads_source/pull/44) introduces the following updates:
+
+## Schema Updates
+
+**9 total changes • 0 possible breaking changes**
+| **Data Model** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| stg_facebook_ads__demographics_country | New Model |   |   |  Uses `demographics_country` source table  |
+| stg_facebook_ads__demographics_country_tmp | New Model |   |   | Uses `demographics_country` source table   |
+| stg_facebook_ads__demographics_country_actions | New Model |   |   |  Uses `demographics_country_actions` source table  |
+| stg_facebook_ads__demographics_country_actions_tmp | New Model |   |   | Uses `demographics_country_actions` source table   |
+| stg_facebook_ads__demographics_region | New Model |   |   |  Uses `demographics_region` source table  |
+| stg_facebook_ads__demographics_region_tmp | New Model |   |   | Uses `demographics_region` source table   |
+| stg_facebook_ads__demographics_region_actions | New Model |   |   |  Uses `demographics_region_actions` source table  |
+| stg_facebook_ads__demographics_region_actions_tmp | New Model |   |   | Uses `demographics_region_actions` source table   |
+| stg_facebook_ads__account_history | New Columns |   | `business_state`, `timezone_offset_hours_utc`, `min_daily_budget`  |  |
+
+## Feature Updates:
+- Added the `facebook_ads__using_demographics_country` and `facebook_ads__using_demographics_region` variables, which can be used to enable or disable the above transformations related to the `demographics_country`/`demograhics_country_actions` and `demographics_region`/`demographics_region_actions` tables.
+  - These variables are dynamically set for Fivetran Quickstart users, but **false** by default otherwise. See [README](https://github.com/fivetran/dbt_facebook_ads_source?tab=readme-ov-file#enable-or-disable-country-and-region-reports) for more details.
```

Comment:
```suggestion
  - These variables are dynamically set for Fivetran Quickstart users, but **false** by default otherwise. See [README](https://github.com/fivetran/dbt_facebook_ads_source?tab=readme-ov-file#enable-or-disable-country-and-region-reports) for more details on how to enable these models, particularly if you are using dbt Core.
```

Reply:
Adjusting myself directly